### PR TITLE
Ohrm 266:Password reset by all users

### PR DESCRIPTION
**Pull request by supunrandunurk**

### CODE REVIEW

File: symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml [hunk 1]
```diff
@@ -7652,6 +7652,107 @@
...
+      <trans-unit id="1914">
+        <source>OrangeHRM Username</source>
+        <target>FR_OrangeHRM Username</target>
+      </trans-unit>
+      <trans-unit id="1915">
+        <source>Work Email or Other Email</source>
+        <target>FR_Work Email or Other Email</target>
+      </trans-unit>
+      <trans-unit id="1916">
+        <source>OR</source>
```
Comment (samanthajayasinghe) [inline symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml]: Shouldn't part of the lang file .. only phrases should add into lang file

File: symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml [hunk 2]
```diff
@@ -7652,6 +7652,107 @@
...
+      <trans-unit id="1921">
+        <source>Please contact HR admin in order to reset the password</source>
+        <target>FR_Please contact HR admin in order to reset the password</target>
+      </trans-unit>
+      <trans-unit id="1922">
+        <source>Work Email Is Not Set. Please Contact HR Admin in Order to Reset the Password</source>
+        <target>FR_Work Email Is Not Set. Please Contact HR Admin in Order to Reset the Password</target>
+      </trans-unit>
+      <trans-unit id="1923">
+        <source>Please Contact HR Admin in Order to Reset the Password</source>
```
Comment (samanthajayasinghe) [inline symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml]: this is same as lang id 1921

**Comment by RajithaKumara** [comment]
Hi @supunrandunurk 
There are some commits that belongs to OHRM-240, and those are already merged in #477. Am I correct?

File: symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml [hunk 3]
```diff
@@ -7652,6 +7652,107 @@
...
+      <trans-unit id="1930">
+        <source>Username or email should be listed:</source>
+        <target>FR_Username or email should be listed:</target>
+      </trans-unit>
+      <trans-unit id="1931">
+        <source>Reset Password</source>
+        <target>FR_Reset Password</target>
+      </trans-unit>
+      <trans-unit id="1932">
+        <source>Enter A New Password</source>
```
Comment (samanthajayasinghe) [inline symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml]: Why upper case?

File: symfony/lib/model/doctrine/orangehrmSecurityAuthenticationPlugin/ResetPasswordLog.class.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+/**
```
Comment (samanthajayasinghe) [inline symfony/lib/model/doctrine/orangehrmSecurityAuthenticationPlugin/ResetPasswordLog.class.php]: Add standard OS header for the class

**Pull request closed without merging**